Taskfile.yml

```diff
@@ -78,6 +79,15 @@ tasks:
           -html={{.COVERPROFILE}} \
           -o coverage.html
         open ./coverage.html
+  helm-install:
+    silent: true
+    cmds:
+      - |
+        if ! helm version | grep -q "{{.HELM_VERSION}}"; then
+          DESIRED_VERSION={{.HELM_VERSION}} curl \
+            -fsSL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 |\
+            bash -s -- --version {{.HELM_VERSION}}
```

Comment:
It looks like you are using both the `DESIRED_VERSION` environment variable and the `--version` flag to make sure the right version gets installed. Do you really need both?

Reply:
Just tested it and you are right that it can be omitted and is not in use by the script.

Reply:
Updated.